mbed_targets/_internal/target_attributes.py

```diff
@@ -30,6 +32,17 @@ class TargetAttributesNotFoundError(TargetAttributesError):
     """Attributes for target not found in targets.json."""
 
 
+@dataclass
```

Comment:
Nice!

Reply:
Do we plan on modifying the attribute values later on? If not it might be worth setting `frozen=True` to make the object immutable.

Reply:
We shouldn't ever be modifying it. I will ❄️ it!

Reply:
ebc4dcf